superbuild/projects_modules/medInria.cmake

```diff
@@ -141,8 +141,10 @@ ExternalProject_Add(${ep}
 ## #############################################################################
 
 if (WIN32)
+  file(REAL_PATH ${TTK_DIR}/../../../ _TTK_DIR)
```

Comment:
I had not seen this when the original PR was merged, but `<package_name>_DIR` variables should no longer be used in the superproject (use `<package_name>_ROOT` only).
`TTK_DIR` is not defined here, and this results in the links not being created in the code below (if you check 
 the `medInria-build\Debug\bin` directory there are no TTK links there).
(Note this does not affect the packaged application. It only prevents running medInria in the build tree)